### ISSUE TITLE
minimal cipher swallows thrown errors in jest

### ISSUE THREAD

**Issue by OR13** [digitalbazaar/edv-client#82]
This line is expected to  throw, and the rest of the test, is mostly  extraneous...
https://github.com/digitalbazaar/edv-client/blob/master/tests/30-EdvStreams.spec.js#L192

```
 const stream = new ReadableStream({
      async pull(controller) {
        // Note: user will call `read` on the decrypt stream... which will
        // trigger a pull here
        if(chunkIndex >= state.chunks) {
          // done
          controller.close();
          return;
        }
         console.log('throws here')
        // get next chunk and enqueue it for reading
        const chunk = await self._getChunk(
          {doc, chunkIndex, capability, invocationSigner});

        console.log('never is hit')
        chunkIndex++;
        controller.enqueue(chunk);
      }
    });
    // const decryptStream = await cipher.createDecryptStream({keyAgreementKey});
    // return stream.pipeThrough(decryptStream);
    return stream

```
 for some reason, the pipeThrough swallows the async exception.... but only in jest
    
Its fine to close these jest related errors as "won't fix" if jest is not intended to be supported... I am will still open them for future searchers :)

**Comment by aljones15** [comment]
I just added help wanted. Our test suite is built in mocha. There might be a jest setting that fixes the issue or maybe it is just the real concurrency jest uses in this case. Maybe you should file the bug on jest's github and see what they think?

**Comment by dlongley** [comment]
Should be addressed by #103.
